xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -719,7 +723,11 @@ bool CVideoPlayer::OpenFile(const CFileItem& file, const CPlayerOptions &options
 
   m_openEvent.Reset();
   Create();
-  CGUIDialogBusy::WaitOnEvent(m_openEvent, g_advancedSettings.m_videoBusyDialogDelay_ms, false);
+  if (!CGUIDialogBusy::WaitOnEvent(m_openEvent, g_advancedSettings.m_videoBusyDialogDelay_ms, true))
```

Comment:
https://github.com/xbmc/xbmc/pull/12792/files#diff-d920d5ea15baa1b7ba5b617ae9b76489R704 uses hardcoded 2 secs for `CGUIDialogBusy::WaitOnEvent`, here you are using advanced setting value. Is there a reason for the different handling?

Reply:
yes, here the busy dialog is expected to come up after a short time. the other one is an exceptional path and as such not configurable.

Reply:
Okay.